contrib/scalar/docs/getting-started.md

```diff
@@ -0,0 +1,112 @@
+Getting Started
+===============
+
+Registering existing Git repos
+------------------------------
+
+To add a repository to the list of registered repos, run `scalar register [<path>]`.
+If `<path>` is not provided, then the "current repository" is discovered from
+the working directory by scanning the parent paths for a path containing a `.git`
+folder, possibly inside a `src` folder.
+
+To see which repositories are currently tracked by the service, run
+`scalar list`.
+
+Run `scalar unregister [<path>]` to remove the repo from this list.
+
+Creating a new Scalar clone using the GVFS Protocol
```

Comment:
Do we need a footnote somewhere saying who does and does not support the GVFS protocol ?

Reply:
Is peer section about scalar cloning without the gvfs protocol ?

Reply:
I added a bit saying that Azure Repos supports the protocol, as an example. We need to be a bit careful because technically BitBucket supports it with an add-on (not by default).